.JuliaFormatter.toml

```diff
@@ -1,11 +1,7 @@
 always_for_in = true
 remove_extra_newlines = true
 always_use_return = true
-whitespace_in_kwargs = true
-whitespace_typedefs = true
-whitespace_ops_in_indices = true
+whitespace_in_kwargs = false
 annotate_untyped_fields_with_any = false
 format_docstrings = true
 conditional_to_if = true
-margin = 92
-separate_kwargs_with_semicolon = true
```

Comment:
Apart from `whitespace_in_kwargs`, the original lines match the styleguide.

Whitespace for typedefs is not mentioned in the styleguide, and is very varied in use, but `true` matches best with the closest examples in the styleguide.

Reply:
The examples I found in the style guide looked like a false for whitespace in type defs

(really I only found this:
```
struct MyArray{T,N} <: AbstractArray{T,N}
    data::AbstractArray{T,N}
    metadata::Dict
end
```)

Reply:
I was thinking more of Unions so I missed that one.

I prefer the space there for readability, and because it is infrequent enough that brevity isn't too important, but either way fits with the overall style so I can cope if you prefer to de-whitespace which I assume you do.

Reply:
on the kwargs I prefer it, on the types, I don't much care either way. The style guide doesn't specifically say and you prefer the whitespace, so I think we can get away with that one 😉